neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -175,6 +178,26 @@ private void OnGetBlocksMessageReceived(GetBlocksPayload payload)
             Context.Parent.Tell(Message.Create(MessageCommand.Inv, InvPayload.Create(InventoryType.Block, hashes.ToArray())));
         }
 
+        private void OnGetFullBlocksMessageReceived(GetFullBlocksPayload payload)
+        {
+            for (uint i = payload.IndexStart; i < payload.IndexStart + payload.Count; i++)
+            {
+                Block block = Blockchain.Singleton.Store.GetBlock(i);
+                if (block == null)
+                    break;
+
+                if (bloom_filter == null)
+                {
+                    Context.Parent.Tell(Message.Create(MessageCommand.Block, block));
+                }
+                else
+                {
+                    BitArray flags = new BitArray(block.Transactions.Select(p => bloom_filter.Test(p)).ToArray());
+                    Context.Parent.Tell(Message.Create(MessageCommand.MerkleBlock, MerkleBlockPayload.Create(block, flags)));
```

Comment:
@shargon and @ixje, if we are really always replying with a bunch of blocks I think that we should use the `zip` tools that @shargon committed some time ago.

If we are replying isolated blocks for a given request, we should random the response, as in this PR here: https://github.com/neo-project/neo/pull/1164

Reply:
What `zip` tools and why do we need to use that? Messages get automatically compressed if their payload exceeds a threshold.
https://github.com/neo-project/neo/blob/4112df22fa5c5080dc0c39469e298db7fa4f85ef/neo/Network/P2P/Message.cs#L35-L37

Reply:
@vncoelho do you have any remaining feedback? I'd like to move this PR forward as it is currently stopping me from implementing syncing logic for `neo3-python`. I prefer to use simplified logic that this PR allows instead of unnecessary complex.

Reply:
No, makes sense this about the message. I had forgot.